database/migrations/2019_01_16_174349_add_actions_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddActionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('actions', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name', 255)->index()->comment('The action name of the post.');
+            $table->integer('campaign_id')->index()->comment('The campaign id the post is associated to.');
+            $table->string('post_type', 255)->index()->comment('The type of post the post is (e.g. photo, text, voter-reg, etc).');
```

Comment:
Do we want to support actions that could have multiple post types? (For example, the old Transfomers campaign where you could reportback on how you beat bullying using either a photo or a text post?)

Reply:
ohh interesting, @DFurnes ! I think I am in the same boat as @katiecrane and think of that as two separate actions?

Reply:
I don't have strong opinions either way, but let's make sure whatever we decide matches with the way the campaigns team will want to use this!

Reply:
Great, point! Thanks for flagging and I will write in my notes to run by campaigns !